### PR TITLE
[JBPM-9764] Test is missing for fix in RHPAM-3712 (empty user when bypass is set)

### PR DESCRIPTION
**Pull request by afalhambra**
**[JBPM-9764](https://issues.redhat.com/browse/JBPM-9764)**: Test is missing for fix in RHPAM-3712 (empty user when bypass is set)

### CODE REVIEW

**Comment by afalhambra** [comment]
Jenkins retest this

**Comment by afalhambra** [comment]
Jenkins retest this

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/ByPassUserTaskServiceIntegrationTest.java [hunk 1]
```diff
@@ -94,4 +94,42 @@ public void testProcessWithUserTasks() {
...
+            processClient.abortProcessInstance(CONTAINER_ID, processInstanceId);
+        }
+    }
+
+    private void assertTaskList(List<TaskSummary> taskList, int size) {
+        assertNotNull(taskList);
+        assertEquals(size, taskList.size());
+        if (taskList.size() > 0) {
+            TaskSummary taskSummary = taskList.get(0);
+            assertEquals("First task", taskSummary.getName());
```
Comment (gmunozfe) [inline kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/ByPassUserTaskServiceIntegrationTest.java:131]: You could pass the name of the task as parameter and refactor other similar asserts in the other test method `testProcessWithUserTasks`, converting task names into constants, wdyt?

**Review by gmunozfe — COMMENTED** [review]
Looks good to me, great job @afalhambra !
Just one suggestion to refactor the other test method and taking advantage of the auxiliary assert method

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/ByPassUserTaskServiceIntegrationTest.java (hunk 1 above)
Comment (afalhambra) [reply kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/ByPassUserTaskServiceIntegrationTest.java]: yes! thanks for the hint!

**Comment by afalhambra** [comment]
timed out - jenkins retest this

**Comment by afalhambra** [comment]
Jenkins retest this

**Comment by afalhambra** [comment]
failing test not related - jenkins retest this

**Comment by afalhambra** [comment]
timed out again
Jenkins retest this

**Comment by afalhambra** [comment]
@mareknovotny please merge this along with backport https://github.com/kiegroup/droolsjbpm-integration/pull/2521

**Pull request merged**